requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy
 accelerate>=0.20.3
 packaging
-transformers>=4.34.1
+transformers>=4.34.1,<4.41
```

Comment:
Wrong place to make this update, should be in `pyproject.yaml`, this file is no longer used for package dependency versions or installation

Reply:
created PR #203 to fix this